File: recyclerViewTopTixed/src/main/java/com/fxffxt/recyclerviewtoptixed/MainActivity.java
package com.fxffxt.recyclerviewtoptixed;

import android.content.Context;
import android.os.Bundle;
import android.util.AttributeSet;
import android.view.LayoutInflater;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        setTheme(R.style.Theme_GitAndroidLibaray);
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        LayoutInflater.from(this).setFactory(new LayoutInflater.Factory() {
            @Nullable
            @Override
            public View onCreateView(@NonNull String name, @NonNull Context context, @NonNull AttributeSet attrs) {
                return null;
            }
        });
    }
}

File: app/src/main/java/com/fxffxt/gitandroidlibaray/MainActivity.kt
package com.fxffxt.gitandroidlibaray

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.fxffxt.gitandroidlibaray.fragmenttest.FirstActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<View>(R.id.tv_inner).setOnClickListener {
            startActivity(Intent(this, FixedInnerActivity::class.java))
        }
        findViewById<View>(R.id.tv_outer).setOnClickListener {
            startActivity(Intent(this, FixedOuterActivity::class.java))
        }
        findViewById<View>(R.id.fragmentTest).setOnClickListener {
            startActivity(Intent(this,FirstActivity::class.java))
        }

    }
}
